code/game/objects/structures/dropship_equipment.dm

```diff
@@ -306,7 +306,116 @@
 		linked_console.selected_equipment = src
 		to_chat(user, span_notice("You select [src]."))
 
+//////////////////////////////////// rappel system //////////////////////////////////////
+/obj/structure/dropship_equipment/rappel_system
```

Comment:
Why is this a dropship equipment type if you treat it as a permanent installation is it dropship equipment or a new installation